po/es.po

```diff
@@ -10529,35 +10530,35 @@ msgstr "reconstrucción de luces"
 
 #: ../src/iop/shadhi.c:866
 msgid "correct shadows"
-msgstr "corrección de sombras"
+msgstr "Corrección de sombras"
 
 #: ../src/iop/shadhi.c:867
 msgid "correct highlights"
-msgstr "corrección de luces"
+msgstr "Corrección de luces"
 
 #: ../src/iop/shadhi.c:868
 msgid "shift white point"
-msgstr "mover punto blanco"
+msgstr "Mover punto blanco"
 
 #: ../src/iop/shadhi.c:870
 msgid "filter to use for softening. bilateral avoids halos"
-msgstr "filtro de suavizado. bilateral evita los halos"
+msgstr "Filtro de suavizado. El bilateral evita los halos"
 
 #: ../src/iop/shadhi.c:871
 msgid ""
 "compress the effect on shadows/highlights and\n"
 "preserve midtones"
 msgstr ""
-"comprimir el efecto en luces/sombras y\n"
-"preservar tonos medios"
+"Comprimir el efecto en luces/sombras\n"
+"preservando tonos medios"
 
 #: ../src/iop/shadhi.c:872
 msgid "adjust saturation of shadows"
-msgstr "ajuste de la saturación de sombras"
+msgstr "Adecuar la saturación de sombras"
```

Comment:
I would rather say "Ajustar la saturación de sombras", _adecuar_ is not a common verb.

Reply:
For the action, it is the corresponding verb.